sensor_msgs/PointCloud2 object_cloud
---
